libimage/pull.go

```diff
@@ -442,8 +442,13 @@ func (r *Runtime) imagesIDsForManifest(manifestBytes []byte, sys *types.SystemCo
 	if err != nil {
 		return nil, fmt.Errorf("listing images by manifest digest: %w", err)
 	}
-	results := make([]string, 0, len(images))
+	results := []string{}
```

Comment:
Can you add a comment on the reasoning and point to containers/podman/issues/18647?